libcloud/test/common/test_aws.py

```diff
@@ -219,6 +225,16 @@ def test_get_payload_hash_returns_digest_of_empty_string_for_GET_requests(self):
         self.assertEqual(self.signer._get_payload_hash(method='GET'),
                          'e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855')
 
+    def test_get_payload_hash_with_data_for_PUT_requests(self):
+        SignedAWSConnection.method = 'PUT'
+        self.assertEqual(self.signer._get_payload_hash(method='POST', data='DUMMY'),
```

Comment:
IS the method still supposed to be POST here? If so, a comment to explain why would be welcome